### DISCUSSION TITLE
Argument of type '{ headers: { Authorization: string; }; }' is not assignable to parameter of type

### DISCUSSION THREAD

**Discussion by lawplatform** [supabase/supabase#19086] [Questions] [answered]
try to use supabse with next-auth I followed this manual
https://authjs.dev/reference/adapter/supabase

but when I init my supabase client. it makes error 

```
const supabase = createClient<Database>(supabaseUrl, supabaseKey, {
				headers: {
					Authorization: `Bearer ${supabaseAccessToken}`,
				},
			});
```
error message is 

Argument of type '{ headers: { Authorization: string; }; }' is not assignable to parameter of type 'SupabaseClientOptions<"public">'.
  Object literal may only specify known properties, and 'headers' does not exist in type 'SupabaseClientOptions<"public"

**Comment by GaryAustin1** [comment, marked as answer]
Your format for options is incorrect.
![image](https://github.com/supabase/supabase/assets/54564956/1dd408bd-9cbd-4f74-a981-965b4ff84137)
https://supabase.com/docs/reference/javascript/initializing?example=with-additional-parameters

**Comment by lawplatform** [comment]
thanks! it solved the issuse :)
